docs/reST/ref/math.rst

```diff
@@ -332,7 +332,14 @@ Multiple coordinates can be set using slices or swizzling
       | :sl:`calculates the angle to a given vector in degrees.`
       | :sg:`angle_to(Vector2) -> float`
 
-      Returns the angle between self and the given vector.
+      Returns the angle from self to the passed ``Vector2`` that would rotate self
+      to be aligned with the passed ``Vector2`` without crossing over the negative 
+      x-axis.
+
+      .. figure:: code_examples/angle_to.png
+         :alt: angle_to image
```

Comment:
This alt text could have been a bit better:
> Shows the angle_to going anti-clockwise from the first "self" vector to the second "other" vector.

tips for good alt text: https://supercooldesign.co.uk/blog/how-to-write-good-alt-text

Reply:
The problem with that characterization is that it is false. It very well could return a clockwise angle as long as that wouldn’t cross the negative x-axis